packages/components/src/components/Image/images.ts

```diff
@@ -5,9 +5,10 @@ export const SVG_IMAGES = {
     UNI_SUCCESS: 'uni-success.svg',
     UNI_WARNING: 'uni-warning.svg',
     T_BRIDGE_CHECK: 't-bridge-check.svg',
-    T_DEVICE_CONFIRM: 't-device-confirm.svg',
+    '1_DEVICE_CONFIRM': 'device-confirm-1.svg',
+    T_DEVICE_CONFIRM: 'device-confirm-t.svg',
+    R_DEVICE_CONFIRM: 'device-confirm-r.svg',
```

Comment:
nit: since it's being renamed
```suggestion
    'DEVICE_CONFIRM_T1': 'device-confirm-t1.svg',
    DEVICE_CONFIRM_TT: 'device-confirm-tt.svg',
    DEVICE_CONFIRM_TR: 'device-confirm-tr.svg',
```

Reply:
good catch!

Reply:
1726c930ef6dd82c556b98f43a049aba7e213994